packages/components/src/Chip/Chip.tsx

```diff
@@ -41,19 +42,21 @@ export const Chip = ({
     label,
     heading,
   );
+  const Tag = onClick ? "button" : "div";
 
   return (
-    <Tooltip message={tooltipMessage}>
-      <button
+    <Tooltip message={tooltipMessage} setTabIndex={false}>
+      <Tag
         className={classes}
-        onClick={(ev: React.MouseEvent<HTMLButtonElement>) =>
-          onClick && onClick(value, ev)
+        onClick={(ev: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) =>
+          onClick?.(value, ev)
         }
         tabIndex={tabIndex}
```

Comment:
If you make the following change, you should be able to fix the issue with disabled chips being currently focusable:
```suggestion
        tabIndex={disabled ? -1 : tabIndex}
```

Reply:
done and done!